data/monitoring/prom-rule.yaml

```diff
@@ -37,3 +37,15 @@ spec:
           for: 5m
           labels:
             severity: warning
+        - expr: sum(up{namespace='{{ .Namespace }}', pod=~'kubemacpool-mac-controller-manager-.*'} or vector(0))
+          record: kubevirt_cnao_kubemacpool_manager_num_up_pods_total
+        - expr: sum(kubevirt_cnao_cr_kubemacpool_deployed{namespace='{{ .Namespace }}'} or vector(0))
+          record: kubevirt_cnao_cr_kubemacpool_deployed_total
+        - alert: KubemacpoolDown
+          annotations:
+            summary: Kubemacpool is deployed by CNAO CR but kubemacpool pod is down.
+            runbook_url: http://kubevirt.io/monitoring/runbooks/KubemacpoolDown
+          expr: kubevirt_cnao_cr_kubemacpool_deployed_total == 1 and kubevirt_cnao_kubemacpool_manager_num_up_pods_total == 0
```

Comment:
maybe due to a bug on CNAO, or just missing kmp on the CR
kubevirt_cnao_cr_kubemacpool_deployed / kubevirt_cnao_cr_kubemacpool_deployed_total will be zero, and then the alert won't raise, it might be fine, but just raising it to be sure

Reply:
the logic is that if KMP is not in the CR then KMP should *not* be deployed at all, hence the alert shouldn't fire (i.e. KMP is down but it's ok, and not something the sysAdmin should wake up in the middle of the night for).